CHANGELOG.asciidoc

```diff
@@ -3,6 +3,112 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.6.0]]
+=== Beats version 7.6.0
+https://github.com/elastic/beats/compare/v7.5.1...v7.6.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Allow Metricbeat's beat module to read monitoring information over a named pipe or unix domain socket. {pull}14558[14558]
+- Remove version information from default ILM policy for improved upgrade experience on custom policies. {pull}14745[14745]
+- Running `setup` cmd respects `setup.ilm.overwrite` setting for improved support of custom policies. {pull}14741[14741]
+- Libbeat: Do not overwrite agent.*, ecs.version, and host.name. {pull}14407[14407]
```

Comment:
This is rather a fix. like "Fix existing agent.*, ecs.version, and host.name fields getting overwritten by Beats if they are already present in the original event".

Reply:
I changed the wording as suggested.